odoo/modules/module.py

```diff
@@ -593,7 +595,7 @@ def run_unit_tests(module_name, position='at_install'):
     threading.currentThread().testing = True
     config_tags = TagsSelector(tools.config['test_tags'])
     position_tag = TagsSelector(position)
-    r = True
+    r = set()
```

Comment:
By the way, we could benefit from this PR to rename this `r` to something more explicit like `report`?